flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -281,6 +281,14 @@ public StateBackend getCheckpointBackend() {
         return checkpointStreamBackend;
     }
 
+    @Override
+    public boolean supportsNoClaimRestoreMode() {
+        // We are able to create full snapshots when requested by
+        // CheckpointOptions#isForcedFullSnapshot. (we might potentially reupload some shared files
```

Comment:
This method `CheckpointOptions#isForcedFullSnapshot` has been dropped in current version of PR.